specification/storagesync/resource-manager/readme.md

```diff
@@ -155,7 +164,7 @@ See configuration in [readme.go.md](./readme.go.md)
 
 See configuration in [readme.java.md](./readme.java.md)
 
-## Multi-API/Profile support for AutoRest v3 generators 
+## Multi-API/Profile support for AutoRest v3 generators
 
 AutoRest V3 generators require the use of `--tag=all-api-versions` to select api files.
 
```

Comment:
IMHO, the multiapi profile should also have the new version.

Reply:
Good point. However, I see that the there is some text there that reads: "This block is updated by an automatic script. Edits may be lost!", so I feel maybe it should have been automatically updated. I'll add the new version there anyways.